bin/exec-env

```diff
@@ -0,0 +1,14 @@
+asdf_update_dotnet_home() {
```

Comment:
needs a shebang, and it's a good idea to add this second line too:

```suggestion
#!/usr/bin/env bash
set -euo pipefail
asdf_update_dotnet_home() {
```

Reply:
Thanks for the feedback. The file is executable.